java-ast/addon/src/main/java/org/jboss/windup/ast/java/ReferenceResolvingVisitor.java

```diff
@@ -1131,17 +991,20 @@ public boolean visit(MethodInvocation node)
         if (resolveTypeBinding != null)
         {
             resolutionStatus = ResolutionStatus.RESOLVED;
-            ITypeBinding[] arguments = resolveTypeBinding.getParameterTypes();
+            ITypeBinding[] argumentTypeBindings = resolveTypeBinding.getParameterTypes();
+            List<Expression> arguments = node.arguments();
 
-            for (ITypeBinding type : arguments)
+            int index = 0;
+            for (ITypeBinding type : argumentTypeBindings)
             {
+                Expression expression = arguments.get(index++);
```

Comment:
Good point... I assume that you can fix this during the merge? Or should I update the PR?